capa/main.py

```diff
@@ -1016,8 +1016,7 @@ def main(argv=None):
             return E_INVALID_FILE_TYPE
 
     try:
-        rules = get_rules(args.rules, disable_progress=args.quiet)
-        rules = capa.rules.RuleSet(rules)
+        rules = capa.rules.RuleSet(get_rules(args.rules, disable_progress=args.quiet))
```

Comment:
mypy has this annoying thing where it doesn't like to see variables get reassigned with different types (tbh, i guess i dont either, in general, but its a common to construct data via refinement) so i've removed the temporary variable here. dont love it.